cmd/shannon.go

```diff
@@ -4,35 +4,31 @@ import (
 	"fmt"
 
 	"github.com/pokt-network/poktroll/pkg/polylog"
+	sdk "github.com/pokt-network/shannon-sdk"
```

Comment:
Not a blocker, but I suggest skipping details related to packages other than `protocol` and `gateway` in the first couple of iterations, to allow more efficient addressing of review comments.